shell/client/styles/_setup-wizard.scss

```diff
@@ -52,6 +52,22 @@
     margin: 4px;
     text-decoration: none;
   }
+
+  .setup-system-log {
+    color: #000000;
+  }
+
+  .setup-system-log-header {
+    display: flex;
```

Comment:
Props for `flex`. I believe this means that if the window is not very tall, then it won't cause unnecessary scrolling.

Let me know if I'm misunderstanding though!